develop/tutorials/articles/194-adaptive-media/00-adaptive-media-images-rest-apis-intro.markdown

```diff
@@ -1,312 +1,365 @@
+404 (Non-existent configuration)
+
+Below is an example response object:
+
+    {
+    "name": "Small demo size",
+    "id": "demo-small",
+    "max-width": "100",
+    "max-height": "100"
+    }
+
+### Add a New Configuration [](id=add-a-new-configuration)
+
+Below is an example request pattern:
+
+    PUT [server address]/o/adaptive-media-rest/images/[configurationId]
+
+**Status Codes**
+
+200<br />
+400 (wrong body)<br />
+403 (user without permission)
+
+Below is an example request:
+
+    PUT [server address]/o/adaptive-media-rest/images/small
+
+    BODY:
+    {
+    "name": "Small Size",
+    "max-height": 100,
+    "max-width": 100
+    }
+
+Here is the resulting configuration:
+
+    {
+    "name": "Small Size",
+    "id": "small",
+    "max-height": "100",
+    "max-width": "100"
+    }
+
+### Delete a Configuration [](id=delete-a-configuration)
+
+Below is an example request pattern:
+
+    DELETE [server address]/o/adaptive-media-rest/images/[configurationId]
+
+**Status Codes**
+
+200<br />
+403 (user without permission)
+
+Below is an example request:
+
+    DELETE [server address]/o/adaptive-media-rest/images/small
+
+    BODY:
+    {
+    "name": "Small Size",
+    "max-height": 100,
+    "max-width": 100
+    }
+
+Here is the resulting deleted configuration:
+
+    {
+    "name": "Small Size",
+    "id": "small",
+    "max-height": "100",
+    "max-width": "100"
+    }
+
+Now that you know how to use the */configuration* endpoint, you can learn how to
+use the */content* endpoint next.
+
+## Content Endpoint [](id=content-endpoint)
+
+To use the */content* endpoint you must first choose the Adaptive Media
+image you want to recover. You can specify an adaptive media image via two
+possible forms of ID: `fileEntryId` or `fileVersionId`.
+<!-- Where can I find this information for the Adaptive Media images? -->
+
+If you use the `fileEntryId`, You’ll need to provide the `fileEntryId` along
+with the version name. The version name is a string (1.0, 1.2, 2.3, etc.).
+<!-- Where can I find the version name? -->
+
+Below is an example request pattern that uses the `fileEntryId`:
+
+  [server address]/o/adaptive-media-rest/images/content/file/**fileEntryId**/version/**version**
+
+You can also use the keyword `last` to retrieve the last version of the file
+entry:
+
+  [server address]/o/adaptive-media-rest/images/content/file/**fileEntryId**/version/last
+
+Using the `fileVersionId` allows you to directly retrieve the Adaptive
+Media image, without the need of a version name:
+
+  [server address]/o/adaptive-media-rest/images/content/**fileVersionId**
+
+Once you've specified a file version, you can use a few additional endpoints to
+get data on the image variants.
+
+You can learn how to use the */config* endpoint next.
+
+### Config Endpoint [](id=config-endpoint)
+
+The */config* endpoint uses a `fileVersion` to retrieve an image generated by a
+certain configuration. If no data is found, the API will return the original
```

Comment:
@ahdezma  Does the `/config` endpoint only accept the `fileEntryId` and not the `fileVersionId` since it uses the file version?